bucket/better-joy.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "6.4b",
+    "description": "Allows the Nintendo Switch Pro Controller, Joycons and SNES controller to be used with CEMU, Citra, Dolphin, Yuzu and as generic XInput",
+    "homepage": "https://github.com/Davidobot/BetterJoy",
+    "license": "MIT",
+    "notes": "How to use: https://github.com/Davidobot/BetterJoy#how-to-use",
+    "url": "https://github.com/Davidobot/BetterJoy/releases/download/v6.4b/BetterJoy_v6.4b.zip",
+    "hash": "8a9212b02eaa2fa016b5e33c11bb1530621d74a3faad43ba9427419943154e8d",
+    "persist":[
+        "BetterJoyForCemu.exe.config"
+    ],
```

Comment:
Does not make sense to persist static file, which is not used.

Pre-create and persist correct settings file

Reply:
`BetterJoyForCemu.exe.config`is the setting file of betterjoy.
see: https://github.com/Davidobot/BetterJoy/wiki/App-Settings

Reply:
After you run it, there is `settings` file created. This config file is not edited by any manipulation with the program.

Reply:
How should I solve this problem? Are there any examples?

Reply:
No good solution has been found at present. So I removed `persist`...